src/libs/ReportActionsUtils.ts

```diff
@@ -511,7 +511,7 @@ function filterOutDeprecatedReportActions(reportActions: ReportActions | null):
  * to ensure they will always be displayed in the same order (in case multiple actions have the same timestamp).
  * This is all handled with getSortedReportActions() which is used by several other methods to keep the code DRY.
  */
-function getSortedReportActionsForDisplay(reportActions: ReportActions | null, shouldMarkTheFirstItemAsNewest = false): ReportAction[] {
+function getSortedReportActionsForDisplay(reportActions: ReportActions | ReportAction[] | null, shouldMarkTheFirstItemAsNewest = false): ReportAction[] {
```

Comment:
will this function work when we pass an array?

Reply:
are we changing it because of test?

Reply:
if yest I would adjust tests to pass correct types instead of changing a type here @fabioh8010 WYDT?

Reply:
Yes it works as expected with array (no diff between arrays and objects, since we are still dealing with key values pairs)